chaco/api.py

```diff
@@ -1,7 +1,167 @@
-""" Defines the publicly accessible items of the Chaco API.
+# (C) Copyright 2006-2021 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+"""
+Defines the publicly accessible items of the Chaco API.
+
+Base
+----
+
+- :attr:`~.NumericalSequenceTrait`
+- :attr:`~.PointTrait`
+- :attr:`~.ImageTrait`
+- :attr:`~.DimensionTrait`
+- :attr:`~.SortOrderTrait`
+- :func:`~.bin_search`
+- :func:`~.reverse_map_1d`
+- :func:`~.right_shift`
+- :func:`~.left_shift`
+- :func:`~.sort_points`
+- :func:`~.find_runs`
+- :func:`~.arg_find_runs`
+- :func:`~.point_line_distance`
+
+Data Model
+----------
+
+- :class:`~.AbstractDataSource`
+- :class:`~.ArrayDataSource`
+- :class:`~.GridDataSource`
+- :class:`~.ImageData`
+- :class:`~.MultiArrayDataSource`
+- :class:`~.PointDataSource`
+- :class:`~.AbstractDataRange`
+- :class:`~.BaseDataRange`
+- :class:`~.DataRange1D`
+- :class:`~.DataRange2D`
+
+
+Mappers
+-------
+
+- :class:`~.AbstractMapper`
+- :class:`~.Base1DMapper`
+- :class:`~.GridMapper`
+- :class:`~.LogMapper`
+- :class:`~.LinearMapper`
+- :class:`~.ColorMapper`
+- :class:`~.ColorMapTemplate`
+- :class:`~.DiscreteColorMapper`
+- :class:`~.TransformColorMapper`
+
+Colormaps and Color Palettes
+----------------------------
+
+- 
```

Comment:
I was actually thinking of making the `*` imports from `chaco.default_colormaps` and `chaco.default_colors` explicit. It will crowd the api module (and the module docstring) but I think it's better than users having to dig through the source code to find what the available color maps are.

And we can move it to the end of the module (and the module docstring) to maintain readability.

Reply:
That makes sense.  I will merge this PR and do that in a follow up PR shortly